scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -176,6 +177,10 @@ fi
 if [[ -n "${2-}" && -n $projectName ]]; then
 	containerIds=$(docker ps --all --filter "name=${projectName}" --quiet)
 	case $2 in
+	"init")
+		echo "initiating project \"${projectName}\"..."
+		create_compose_files
+		;;
```

Comment:
I couldn't get this to work - can you check the logic to activate it?  I was expecting this to work: `./cht-docker-compose.sh new-project.env init`

However I got this:

```
./cht-docker-compose.sh new-project.env init

File "new-project.env" doesnt exist - be sure to include ".env" at the end!
```

If you're gong to keep this logic, please add matching output for `show_help_existing_stop_and_destroy()` [helper function](https://github.com/medic/cht-core/blob/7e148852ea4b2a7a0c36438d789c3bff7a675d09/scripts/docker-helper-4.x/cht-docker-compose.sh#L80).

Reply:
the idea it to have the new-project.env file already created (copy from prod or a destroyed cht) then run 

`./cht-docker-compose.sh new-project.env  init`